apps/endatix-hub/package.json

```diff
@@ -29,7 +29,9 @@
     "sonner": "^1.5.0",
     "tailwind-merge": "^2.5.2",
     "tailwindcss-animate": "^1.0.7",
-    "zod": "^3.23.8"
+    "zod": "^3.23.8",
+    "survey-core": "latest",
```

Comment:
Latest is great, but is dangerous as it will have different outcomes based on different time someone hit `pnpm install`. 

Dobri's PR has fixed versions. We need to use his version for this one once we resolve the conflicts. Sharing this FYI

Reply:
Noted.